notebooks/7-rotation-speeds/interm_rotation_rate.ipynb

```diff
@@ -0,0 +1,2663 @@
+{
```

Comment:
<p>Don't forget sections at the bottom with more info (gah can't remember what it's called) but it includes your name for credit!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='100'/>

Reply:
<p>You can copy/paste this into the Notebook. Note that to set up the "top" shortcut, you'll have to add an html tag at the top of the NB</p><p> </p><p>## About this Notebook</p><p>**Author:** You, if you want credit</p><p>**Last updated:** July 2021 &lt;br&gt;</p><p> </p><p>For support, please contact the Archive HelpDesk at archive@stsci.edu.</p><p> </p><p>***</p><p> &lt;img style="float: right;" src="https://raw.githubusercontent.com/spacetelescope/notebooks/master/assets/stsci_pri_combo_mark_horizonal_white_bkgd.png" alt="Space Telescope Logo" width="200px"/&gt;</p><p> </p><p>[Return to top of page](#top)</p>